extensions/functions/src/main/java/io/crate/operation/aggregation/HyperLogLogDistinctAggregation.java

```diff
@@ -288,8 +290,8 @@ public HllState initialState(RamAccounting ramAccounting, MemoryManager memoryMa
         }
 
         @Override
-        public void loadDocValues(LeafReader reader) throws IOException {
-            values = FieldData.toString(DocValues.getSortedSet(reader, columnName));
+        public void loadDocValues(LeafReaderContext reader) throws IOException {
```

Comment:
```suggestion
        public void loadDocValues(LeafReaderContext leafReaderContext) throws IOException {
```

Reply:
As you like, just felt a bit weird to have `reader.reader()`, but it's really minor, no need to change.